modules/hcp-import-limitations.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-import.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-import-limitations_{context}"]
+= Limitations of managing imported hosted clusters
+
+Hosted clusters are automatically imported into the local {mce}, unlike a standalone {product-title} or third party clusters. Hosted clusters run some of their agents in the _hosted mode_ so that the agents do not use the resources of your cluster. If you choose to automatically import hosted clusters, you can update node pools and the control plane in hosted clusters by using the `HostedCluster` resource on the management cluster. To update node pools and a control plane, see "Updating node pools in a hosted cluster" and "Updating a control plane in a hosted cluster".
+
+You can import hosted clusters into a location other than the local {mce-short} by using the {mce-short} or {rh-rhacm-first}. You must use the following options to update hosted clusters:
```

Comment:
By following https://github.com/stolostron/rhacm-docs/blob/2.12_stage/mce_acm_integration/discover_hosted/acm_discover_hosted.adoc, you  can import hosted clusters into a location other than the local {mce-short} by using the {rh-rhacm-first}. In this topology (scenario), the hosted clusters must be updated through the CLI or the console of the local MCE where the cluster is hosted. You cannot update the hosted clusters through the hub ACM.

Reply:
This paragraph includes line 13. Remove line 13.